pkg/interactive/session_details_type.go

```diff
@@ -0,0 +1,45 @@
+package interactive
+
+import (
+	"fmt"
+
+	"golang.org/x/text/cases"
+	"golang.org/x/text/language"
+
+	"github.com/apono-io/apono-cli/pkg/clientapi"
+	listselect "github.com/apono-io/apono-cli/pkg/interactive/inputs/list_select"
+	"github.com/apono-io/apono-cli/pkg/styles"
+)
+
+func RunSessionDetailsTypeSelector(session *clientapi.AccessSessionClientModel) (string, error) {
+	if len(session.ConnectionMethods) == 0 {
+		return "", fmt.Errorf("no available connection methods")
+	}
+
+	if len(session.ConnectionMethods) == 1 {
+		return session.ConnectionMethods[0], nil
+	}
```

Comment:
I would expect this to be outside the interactive module, because it should used for both interactive and non-interactive modes